test/api/container/JobContainerTest.php

```diff
@@ -48,4 +49,14 @@ public function testGcsOutputPath()
         $this->assertEquals('test_prefix/test/', $jobContainer->getOutputPath());
     }
 
+    public function testLiveStreamJobConfig()
+    {
+        $jobContainer = new JobContainer();
+        $jobContainer->job = new LiveStreamJobConfig();
+
+        $this->assertEquals($jobContainer->job instanceof LiveStreamJobConfig, true);
```

Comment:
This tests if an instance of the new LiveStreamJobConfig can be created and instanceof produces the expected result on it.